src/Uno.UI/UI/Xaml/Application.macOS.cs

```diff
@@ -76,7 +78,17 @@ private ApplicationTheme GetDefaultSystemTheme()
 			else
 			{
 				return ApplicationTheme.Dark;
-			}			
+			}
+		}
+
+		private void SetCurrentLanguage()
+		{
+			try
+			{
+				var language = NSLocale.PreferredLanguages[0];
+				CultureInfo.CurrentUICulture = CultureInfo.CreateSpecificCulture(language);
+			}
+			catch (Exception) { }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove or fill this block of code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5521872)